tough/src/lib.rs

```diff
@@ -628,7 +663,7 @@ fn load_root<R: Read>(
     //    that the expiration of the trusted root metadata file does not matter, because we will
     //    attempt to update it in the next step.
     let mut root: Signed<Root> =
-        serde_json::from_reader(root).context(error::ParseTrustedMetadataSnafu)?;
+        serde_json::from_slice(root.as_ref()).context(error::ParseTrustedMetadataSnafu)?;
```

Comment:
I guess parsing JSON can take a non-negligible amount of time warranting `spawn_blocking`. On the other hand, it's not worse than before, so let's wait and see if they have something to say about it.